db/migrate/20201204232014_remove_matching_state_from_ucas_matches.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveMatchingStateFromUCASMatches < ActiveRecord::Migration[6.0]
```

Comment:
Do we also need to add a migration to update UCAS Matches which are currently in the processed state to either `resolved_on_apply` or `resolved_on_ucas`? Wondering as a lot of them will start flagging as "Action needed" and there we don't have a manual process to mark them as resolved.
I realise it will be difficult to determine where they got resolved 🤔 Especially as for instance we had matches in the previous cycle where statuses were
- RECRUITED on UCAS 
- PENDING CONDITIONS on Apply
They got marked "processed" as the candidates were recruited already there was no point in us contacting them at that stage.

However, how we resolve the matches can impact the statistics. Probably something to discuss with the Transition team.

Reply:
PR that introduces `manually_processed` as per our discussion: 
https://github.com/DFE-Digital/apply-for-teacher-training/pull/3616